topic/timeseries/test.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from testbook import testbook
+
+
+def test_notebook(notebook):
+    """
+    Execute Jupyter Notebook, one test case per .ipynb file.
+    """
+    if notebook.name == "dask-weather-data-import.ipynb":
+        raise pytest.skip("Depends on DOWNLOAD_PATH/daily_weather.parquet")
```

Comment:
Needs to be addressed on a subsequent iteration.

Reply:
It is more than 200MB - I would rather keep this with Kaggle. Maybe we could add the download to the notebook itself. But the test to frequently import 200MB (which results in much more data in CrateDB due to row and columnar storage and indexing) - tbd if we really want to do this frequently, although I really like the idea to have it tested.

Reply:
Can you tell me about the source of the dataset? Is it referenced inside the notebook?

Reply:
Deferred this to a different patch.

- GH-352